nautobot/core/testing/api.py

```diff
@@ -185,13 +187,27 @@ def test_options_object(self):
             self.assertHttpStatus(response, status.HTTP_200_OK)
 
     class ListObjectsViewTestCase(APITestCase):
-        brief_fields = []
         choices_fields = None
         filterset = None
 
         def get_filterset(self):
             return self.filterset or lookup.get_filterset_for_model(self.model)
 
+        def get_depth_fields(self):
+            """Get a list of model fields that could be tested with the ?depth query parameter"""
+            depth_fields = []
+            for field in self.model._meta.fields:
+                if not field.name.startswith("_"):
+                    if isinstance(field, (ForeignKey, GenericForeignKey, ManyToManyField)) and (
+                        field.related_model != ContentType
+                        # user is a model field on Token but not a field on TokenSerializer
+                        and not (field.name == "user" and self.model == users_models.Token)
+                    ):
+                        depth_fields.append(field.name)
+                    elif field.name == "tags":
+                        depth_fields.append(field.name)
```

Comment:
Would it be possible/better to include `TaggableManager` in the `isinstance()` check above instead?